src/MoBi.UI/Views/EditIndividualAndExpressionConfigurationsView.cs

```diff
@@ -130,18 +161,23 @@ public void BindTo(IndividualSelectionDTO individualSelectionDTO)
       public void AddUnusedExpression(ITreeNode treeNodeToAdd)
       {
          projectExpressionsTree.TreeView.AddNode(treeNodeToAdd);
-         projectExpressionsTree.TreeView.SelectNode(treeNodeToAdd);
+         projectExpressionsTree.TreeView.Selection.Add(treeListNodeFor(projectExpressionsTree.TreeView, treeNodeToAdd));
       }
 
       public void AddUsedExpression(ITreeNode treeNodeToAdd)
       {
          simulationExpressionsTree.AddNode(treeNodeToAdd);
-         simulationExpressionsTree.SelectNode(treeNodeToAdd);
+         simulationExpressionsTree.Selection.Add(treeListNodeFor(simulationExpressionsTree, treeNodeToAdd));
+      }
+
+      private TreeListNode treeListNodeFor(UxTreeView treeView, ITreeNode treeNodeToAdd)
```

Comment:
yeah I just check the code in core. There is a method for this as mentioned

```
  public ITreeNode NodeFrom(TreeListNode node)
      {
         return node?.Tag as ITreeNode;
      }

      public TreeListNode NodeFrom(ITreeNode node)
      {
         return node == null ? null : treeListNodeById(node.Id);
      }
```

Reply:
What happend, I thought I had that change. Maybe I undid it somehow

Reply:
You did for the one way but not the other

Reply:
The stuff with single/first. There is a method for this in the tree that finds by id

Reply:
I think I pushed, then requested review, but the push failed.

That first stuff is all gone now.